packages/components/src/components/map-marker/map-marker.ts

```diff
@@ -83,7 +83,8 @@ export default class SdMapMarker extends SolidElement {
         target=${ifDefined(isLink ? this.target : undefined)}
         @blur=${this.handleBlur}
         @focus=${this.handleFocus}
-        aria-labelledby="content"
+        role=${ifDefined(!this.notInteractive ? (isLink ? undefined : 'button') : undefined)}
```

Comment:
I suggest we avoid nested ternary expressions

```suggestion
        role=${ifDefined(this.notInteractive || isLink ? undefined : 'button')}
```

Reply:
Great catch, thanks :)